smoke-test/src/main/java/org/opennms/smoketest/containers/OpenNMSContainer.java

```diff
@@ -166,9 +171,13 @@ public OpenNMSContainer(StackModel model, OpenNMSProfile profile) {
                 .mapToInt(Map.Entry::getValue)
                 .toArray();
 
-        String javaOpts = "-Xms2048m -Xmx2048m -Djava.security.egd=file:/dev/./urandom -javaagent:/opt/opennms/agent/jacoco-agent.jar=output=none,jmx=true";
+        String javaOpts = "-Xms2048m -Xmx2048m -Djava.security.egd=file:/dev/./urandom";
```

Comment:
not sure we should pass `javaOpts` here. This may conflict with other settings in circeci profile. May be @RangerRick can throw some light on this

Reply:
I want to think of ways we can deduplicate this type of configuration, but I'd like to avoid trying to solve that in this PR for now. For example, we configure the jacoco agent in a few places:
```
$ git grep -- '-javaagent:.*jacoco-agent.jar' | wc -l
       4
```

Maybe this is something we can think about as we do container improvements (where we don't use the opennms startup script, we do things slightly differently here for smoke tests vs. other ways options are set in the entrypoint, etc.).